### PR TITLE
Set Up React Application

### PR DESCRIPTION
**Pull request by JaimeCrz**
## Changes proposed in this pull request:
* Setting up / Scaffolding React Application using the typescript template

## Packages added
- Cypress
- Jest
- Babel for typescript
- Other dependencies installed with create-react-app

## (Optional) What I have learned working on this feature:
*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
```
Comment (kaylawoodbury) [inline README.md]: Lets clear this out so we can add our own info later

File: package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "typescript": "^4.1.2",
+    "web-vitals": "^1.0.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
```
Comment (kaylawoodbury) [inline package.json]: was there a jest script here?

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
```
Comment (kaylawoodbury) [inline public/index.html]: can probably remove these comments

**Review by kaylawoodbury — APPROVED** [review]
just a couple comments that can be removed, otherwise it looks good!

File: README.md (hunk 1 above)
Comment (JaimeCrz) [reply README.md]: +1 done =)

File: package.json (hunk 2 above)
Comment (JaimeCrz) [reply package.json]: pushed now =)

File: public/index.html (hunk 3 above)
Comment (JaimeCrz) [reply public/index.html]: Resolved!

**Review by kaylawoodbury — APPROVED** [review]
Looks good!

**Pull request merged**